supports-and-hinges/relconnectsrigidlink.md

```diff
@@ -2,379 +2,38 @@
+|              Name             |      String      |                                                                                                       RL1                                                                                                      |                                   yes                                  | Human readable unique name of the object                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+|             Nodes             |      String      |                                                                                                     N3; N4                                                                                                     |                                   yes                                  | <p>The name of the valid existing node (<a href="../structural-analysis-elements/structuralpointconnection.md#node">StructuralPoinConnection</a>), or (<a href="../structural-analysis-elements/structuralcurvemember.md#1d-member-beam-column">StructuralCurveMember</a>, <a href="../structural-analysis-elements/structuralsurfacemember.md#2d-member-plate-wall">StructuralSurfaceMember</a>) for internal nodes</p><p>Maximum of two nodes can be connected by RelConnectsRigitLink</p><p>First in node is seen as "master node", second in row is "slave node"</p> |
+|         Hinge position        |       Enum       |                                                          <p>None</p><p></p><p>Begin (master node)</p><p></p><p>End (slave node)</p><p></p><p>Both</p>                                                          |                                   yes                                  | <p>This attribute serves to indicate if hinges should be applied to rigid link and if so, than on which node.</p><p>Hinge position allows user to define constrains of selected node or nodes of RelConnectsRigidLink for translation (ui) and rotation (fii)</p>                                                                                                                                                                                                                                                                                                        |
```

Comment:
Example value for the enum is not supported by the SDK!
it's only Begin, End or Both, without the (master node) or (slave node) suffixes

Import might work but export will always be without the suffixes.
If you need this, raise an issue on the SDK to add additional translations for this